src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/CNNVariantScore.java

```diff
@@ -60,32 +63,36 @@
     private static final int REF_INDEX = 2;
     private static final int ALT_INDEX = 3;
     private static final int KEY_INDEX = 4;
+    private static final int FIFO_STRING_CAPACITY = 1024;
 
     @Argument(fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
             shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
             doc = "Output file")
     private String outputFile = null; // output file produced by Python code
 
-    @Argument(fullName = "architecture", shortName = "a", doc = "Neural Net architecture and weights hd5 file", optional = false)
+    @Argument(fullName = "architecture", shortName = "architecture", doc = "Neural Net architecture configuration json file", optional = false)
     private String architecture = null;
```

Comment:
Now the convention is that both files have the same name and different extensions and live in the same directory as we do with bam or vcf index files.  My concern about providing both as arguments is that users could mistakenly supply files that were not in sync with eachother.  Do you think it is worth that risk?  Another alternative is to add a dataset directly into the keras model hd5, but this could backfire if keras changes their internal model serialization...

Reply:
That sounds like a step in the right direction. And yeah we need to ensure somehow that the files are in sync or some way to validate that it doesn't happen. We don't necessarily have to fully address this now. Having said that, is it possible to create your own hd5 file and have keras serialize into a separate group or dataset in that ?